ocaml/xenopsd/lib/xenopsd.ml

```diff
@@ -49,6 +49,8 @@ let default_vbd_backend_kind = ref "vbd"
 
 let ca_140252_workaround = ref false
 
+let xen_platform_pci_bar_uc = ref true
```

Comment:
The default should be false, see https://github.com/xapi-project/xen-api/pull/6591.

Reply:
I can rebase my PR on top of this one and change the default though.